.dockerignore

```diff
@@ -1 +1,2 @@
 .db/
+media/
```

Comment:
Hi @szabozoltan69 

In my local, I am also seeing these files. Add them as well if this exists on yours.
```
.mypy_cache
.pytest_cache
go-logs
go.log
```